site/content/docs/5.0/forms/checks-radios.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: docs
-title: Checks
+title: Checks and radios
```

Comment:
Needs a redirect

Reply:
is redirect something i can do / add to the PR itself? or something that needs to be done separately?

Reply:
You should do it here because that's where you change the URL. Search the content files for `aliases`.

Reply:
I meant "here" as in in the repo (compared to defining the redirects somewhere else in some server configuration file outside of the repo or on some web-based admin interface). But looking at `aliases` now, I think I get it. made an attempt, seems to work ok locally.